### DISCUSSION TITLE
git push gives error 403

### DISCUSSION THREAD

**Discussion by njavali** [community/community#57956] [New to GitHub] [answered]
Hello,
I have created PAT and want to git push commits in order to create pull request.
However I get below error,
remote: Permission to njavali/open-iscsi.git denied to njavali.
fatal: unable to access 'https://github.com/njavali/open-iscsi.git/': The requested URL returned error: 403

Any other config is missing here causing permission denial.
Please assist.

Thanks!

**Comment by airtower-luna** [comment]
That error code indicates the credentials you're using don't allow you to push. There could be multiple reasons, like an expired token or missing permissions. The token you use must have the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).

**Reply by Chava-Cohen** [reply]
אני מקבלת הודעת שגיאה של 403 שהחשבון הושה בעת ביצוע הפעולה PUSH
C:\Users\User\Desktop\חדש בשביל הגיט\Our-project>git push --set-upstream origin Base_WinTech
remote: Your account is suspended. Please visit https://support.github.com for more information.
fatal: unable to access 'https://github.com/Chava-Cohen/WinTech.git/': The requested URL returned error: 403

**Reply by airtower-luna** [reply]
> Your account is suspended

Can't tell you why (you'd have to ask support) but that's your problem right there.

**Comment by njavali** [comment, marked as answer]
Thank you very much for the pointers.

Below Repository permissions helped resolve the error,

Read and Write access to actions, code, commit statuses, pull requests, and workflows.

Thanks once again !!

**Comment by Lupiya** [comment]
GitHub no longer supports https for pushing repositories. Use ssh
